### PR TITLE
use sdk 1.0.0-rc4

### PR DESCRIPTION
**Pull request by enricosada**
with sdk rc4 released, no need of nuget dev feeds

i build and tested locally, and VSCode intellisense works for server.

### CODE REVIEW

**Comment by forki** [comment]
It's red. Is that related?

**Comment by enricosada** [comment]
dont think so, npm errors

```
ERR! Error: EPERM: operation not permitted, rename 'C:\Users\appveyor\AppData\Roaming\npm-cache\lodash\4.17.4\package.tgz.3658517057' -> 'C:\Users\appveyor\AppData\Roaming\npm-cache\lodash\4.17.4\package.tgz'
npm ERR!     at Error (native)
npm ERR!  { [Error: EPERM: operation not permitted, rename 'C:\Users\appveyor\AppData\Roaming\npm-cache\lodash\4.17.4\package.tgz.3658517057' -> 'C:\Users\appveyor\AppData\Roaming\npm-cache\lodash\4.17.4\package.tgz']
npm ERR!   errno: -4048,
npm ERR!   code: 'EPERM',
npm ERR!   syscall: 'rename',
npm ERR!   path: 'C:\\Users\\appveyor\\AppData\\Roaming\\npm-cache\\lodash\\4.17.4\\package.tgz.3658517057',
npm ERR!   dest: 'C:\\Users\\appveyor\\AppData\\Roaming\\npm-cache\\lodash\\4.17.4\\package.tgz',
npm ERR!   parent: 'babel-core' }
npm ERR! 
npm ERR! Please try running this command again as root/Administrato
```

**Comment by forki** [comment]
Wtf. @alfonsogcnunez

Am 16.02.2017 6:05 nachm. schrieb "Enrico Sada" <notifications@github.com>:

**Comment by enricosada** [comment]
green @forki, temporary issue

File: src/Server/Server.fsproj [hunk 1]
```diff
@@ -1,31 +1,24 @@
-﻿<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk" ToolsVersion="15.0">
```
Comment (forki) [inline src/Server/Server.fsproj]: did you test if paket needs that?

Comment (forki) [inline src/Server/Server.fsproj]: as expected paket update now thinks it's a old MsBuild -> need to fix paket

Comment (forki) [inline src/Server/Server.fsproj]: ok fixed that. Paket works with that

**Pull request merged**